dts/msm8974/rules.mk

```diff
@@ -2,4 +2,6 @@ LOCAL_DIR := $(GET_LOCAL_DIR)
 
 DTBS += \
 	$(LOCAL_DIR)/msm8974-lge-d855.dtb \
+	$(LOCAL_DIR)/msm8974-lge-hammerhead-d820.dtb \
+	$(LOCAL_DIR)/msm8974-lge-hammerhead-d821.dtb \
```

Comment:
D821 loads ``msm8974-lge-hammerhead-d820.dtb`` instead of ``msm8974-lge-hammerhead-d821.dtb``, and the possible solution could be changing the order or deleting d820 dtb, which means these two dtbs can't exist at the same time.
[d821.dts.txt](https://github.com/msm8916-mainline/lk2nd/files/7168420/d821.dts.txt)